test/common/typecheck

```diff
@@ -53,7 +53,6 @@ ignored_codes = [
     "TS7005",   # Variable '*' implicitly has an 'any[]' type.
     "TS7006",   # Parameter '*' implicitly has an 'any' type.
     "TS7008",   # Member '*' implicitly has an 'any[]' type.
-    "TS7009",   # 'new' expression, whose target lacks a construct signature, implicitly has an 'any' type.
```

Comment:
I think it's okay to remove this here as well. The problem is not the implicit "any", but that TS missed that something is a constructor, which is worth fixing even in relaxed code.

Reply:
Right, with that new approach we can and should fix similar instances